modules/hal_nordic/nrfx/nrfx_glue.h

```diff
@@ -317,12 +317,14 @@ void nrfx_busy_wait(uint32_t usec_to_wait);
 /** @brief Bitmask that defines DPPI channels that are reserved for use outside of the nrfx library. */
 #define NRFX_DPPI_CHANNELS_USED   (NRFX_PPI_CHANNELS_USED_BY_BT_CTLR |    \
 				   NRFX_PPI_CHANNELS_USED_BY_802154_DRV | \
-				   NRFX_PPI_CHANNELS_USED_BY_MPSL)
+				   NRFX_PPI_CHANNELS_USED_BY_MPSL | \
+				   NRFX_PPI_CHANNELS_USED_BY_NRFE)
```

Comment:
nit: I guess it would be easier to maintain and `git blame` if it was kept in another format:
```
#define NRFX_DPPI_CHANNELS_USED   (NRFX_PPI_CHANNELS_USED_BY_BT_CTLR    \
				   | NRFX_PPI_CHANNELS_USED_BY_802154_DRV \
				   | NRFX_PPI_CHANNELS_USED_BY_MPSL \
				   | NRFX_PPI_CHANNELS_USED_BY_NRFE \
				  ) 
```
It applies to other similar lists.